java/integTest/src/test/java/glide/SharedClientTests.java

```diff
@@ -47,8 +47,14 @@ public static void init() {
         clusterClient =
                 GlideClusterClient.createClient(commonClusterClientConfig().requestTimeout(10000).build())
                         .get();
-
         clients = List.of(Arguments.of(standaloneClient), Arguments.of(clusterClient));
+        assertTrue(!clusterClient.getStatistics().isEmpty());
+        assertEquals(
```

Comment:
why asserting number of the stats entries inside of init? should be on its own test

Reply:
This is where we construct the clients, I figured its a good place to check...